shared/src/main/java/com/couchbase/lite/AbstractReplicator.java

```diff
@@ -560,18 +563,21 @@ private void _start() {
         boolean pull = isPull(config.getReplicatorType());
         boolean continuous = config.isContinuous();
 
-        if (config.getPushFilter() != null || config.getPullFilter() != null) {
-            C4ReplicationFilter filter = new C4ReplicationFilter() {
-                @Override
-                public boolean validationFunction(final String docID, final int flags, final long dict, final boolean isPush, final Object context) {
-                    final AbstractReplicator replicator = (AbstractReplicator) context;
-                    return replicator.validationFunction(docID, flags, dict, isPush);
-                }
-            };
+        if (config.getPushFilter() != null) c4ReplPushFilter = new C4ReplicationFilter() {
+            @Override
+            public boolean validationFunction(final String docID, final int flags, final long dict, final boolean isPush, final Object context) {
+                final AbstractReplicator replicator = (AbstractReplicator) context;
+                return replicator.validationFunction(docID, flags, dict, isPush);
+            }
+        };
 
-            if (config.getPushFilter() != null) c4ReplPushFilter = filter;
-            if (config.getPullFilter() != null) c4ReplPullFilter = filter;
-        }
+        if (config.getPullFilter() != null) c4ReplPullFilter = new C4ReplicationFilter() {
+            @Override
+            public boolean validationFunction(final String docID, final int flags, final long dict, final boolean isPush, final Object context) {
```

Comment:
This line is very long. Please try to limit one line to 100 chars max. I think you can set the limit to Android Studio and Android Studio will show a vertical line.